purview/models/koji.py

```diff
@@ -55,3 +55,4 @@ class KojiTag(PurviewStructuredNode):
     builds = RelationshipTo('KojiBuild', 'CONTAINS')
     id_ = UniqueIdProperty(db_property='id')
     name = StringProperty(required=True)
+    products = RelationshipTo('.product.Product', 'ATTACHED_TO')
```

Comment:
The `RelationshipTo` should likely be on the `Product` class because that is where the relationship is defined.